src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/CollectionRuleDefaultsOptionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options;
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon.Options
+{
+    internal static partial class CollectionRuleDefaultsOptionsExtensions
+    {
+        public static CollectionRuleDefaultsOptions CreateCollectionRuleDefaults(this RootOptions rootOptions, Action<CollectionRuleDefaultsOptions> callback = null)
```

Comment:
```suggestion
        public static RootOptions AddCollectionRuleDefaults(this RootOptions rootOptions, Action<CollectionRuleDefaultsOptions> callback = null)
```

None of the callers currently use the returned value; instead, return `RootOptions` to allow chaining of method calls.

Reply:
I'll change this - but do you want me to actually chain this call with the other set-up, or is this just so we _could_ in the future? I followed the model we use in some of the other tests where we set up `CollectionRules` and `FileSystemEgress` separately, but if you'd prefer I can chain them all together - whatever you'd consider more readable.